tools/src/format.rs

```diff
@@ -18,9 +18,28 @@ use git2;
 use chrono::naive::datetime::NaiveDateTime;
 use chrono::offset::fixed::FixedOffset;
 use chrono::datetime::DateTime;
+use linkify::{LinkFinder, LinkKind};
 
 use config;
 
+fn linkify(s: String) -> String {
```

Comment:
It might be nice to have this in its own module along with the linkify_commit_header function from blame.rs. Maybe we could combine those two together as well into a single linkification function.